sdk/src/main/java/io/dapr/client/domain/QueryStateRequest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2021 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.client.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.dapr.client.domain.query.Query;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class QueryStateRequest {
+
+  @JsonIgnore
+  private final String storeName;
+
+  private Query query;
+
+  private String queryString;
```

Comment:
@artursouza I have added both query and queryString as an option to either build query on the fly or read from file and use in the request. 

Let me know if that is ok.

Reply:
LGTM. I really like the query class.